samples/bluetooth/channel_map_updates/README.rst

```diff
@@ -0,0 +1,280 @@
+.. _bluetooth_channel_map_updates:
+
+Bluetooth: Autonomous Channel Map Updates
+##############################
+
+
+The Channel Map Updates sample demonstrates an autonomous channel map update algorithm for Bluetooth® Low Energy connections using Quality of Service (QoS) events.
+The algorithm automatically evaluates channel performance and generates optimized channel maps to improve connection reliability.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+See the supported boards listed in ``samples/bluetooth/channel_map_updates/sample.yaml``.
+
+You need at least two development kits to physically test this sample:
+
+* This sample currently supports channel map updates to one peripheral device. To demonstrate it, you need two development kits.
+
+You can mix different development kits from the list above.
+
+The sample also requires a connection to a computer with a serial terminal for each of the development kits.
+
+Overview
+********
+
+The sample implements a channel map update algorithm that uses connection event reports to collect Quality of Service data.
+It automatically evaluates channel performance based on CRC errors and RX timeouts, then generates optimized channel maps to improve connection reliability.
+
+The sample can operate in two roles:
+
+* **Central (tester)**: Initiates connections and collects QoS statistics.
+* **Peripheral (peer)**: Accepts connections and participates in the channel evaluation process.
+
+Algorithm Description
+=====================
+
+Rating Calculation
+-------------------
+
+The algorithm evaluates each channel after collecting a configurable number of samples using the following formula:
+
+.. code-block:: text
```

Comment:
Use .. math:: directive to write formula instead of .. code-block::. See https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/libraries/others/emds.rst?plain=1 for ref.